### PR TITLE
[ZEPPELIN-5766] Window compilation (.appveyor.yml)

### PR DESCRIPTION
**Pull request by jooyounghun**
What is this PR for?
Add build script to appveyor.yml for build test

What type of PR is it?
Improvement

Todos
Nothing

What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-5766

How should this be tested?
It is automatically executed while performing CI work on PR
Questions:
Does the licenses files need to update? No
Is there breaking changes for older versions? No
Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@jooyounghun You can just make force commit to trigger the CI, no need to create another PR.

File: zeppelin-web/karma.conf.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ module.exports = function(config) {
       'bower_components/ace-builds/src-noconflict/keybinding-emacs.js',
       'bower_components/ace-builds/src-noconflict/ext-language_tools.js',
       'bower_components/ace-builds/src-noconflict/theme-chrome.js',
+      'bower_components/ace-builds/src-noconflict/mode-javascript.js',
```
Comment (jongyoul) [inline zeppelin-web/karma.conf.js:70]: I saw similar changes in another PR #4424 so you guys need to decide who will add it. :-) @HYEEWON

Comment (HYEEWON) [reply zeppelin-web/karma.conf.js:70]: It doesn't matter who adds it. But, I added it first, so I want to add it. Is it okay?

Comment (jooyounghun) [reply zeppelin-web/karma.conf.js]: I also think the file was generated when building on window. But I don't know why the file is auto-generated. First of all, I pushed another commit with mode-javascript.js removed.

Comment (HYEEWON) [reply zeppelin-web/karma.conf.js]: Thank you :)

Comment (jongyoul) [reply zeppelin-web/karma.conf.js]: Thank you for taking this issue.

**Comment by zjffdu** [comment]
Do you have plan to enable test in windows? @jooyounghun

**Comment by jooyounghun** [comment]
I don't know how to enable test in windows yet, but I will try.

**Pull request merged**